### PR TITLE
Load the app in production

### PR DESCRIPTION
**Pull request by hennevogel**
Make sure we can boot the app in production.

### CODE REVIEW

File: .circleci/conditional_config.yml [hunk 1]
```diff
@@ -155,6 +156,9 @@ jobs:
             cd src/api;
             bundle exec rake dev:lint:rubocop:all
       - *save_rubocop_cache
+      - run:
+          name: Test loading the App in production
+          command: RAILS_ENV=production bundle exec rake db:setup zeitwerk:check
```
Comment (dmarcoux) [inline .circleci/conditional_config.yml:161]: What is the point of doing `db:setup`? `zeitwerk:check` doesn't need the database to ensure classes are loaded properly.

Comment (hennevogel) [reply .circleci/conditional_config.yml]: it does. the check some after the check for the DB...

**Review by eduardoj — APPROVED** [review]

File: .circleci/conditional_config.yml [hunk 2]
```diff
@@ -336,7 +337,18 @@ jobs:
           name: Run spider
           command: |
             cd src/api
-            bundle exec rake test:spider
+            case $CIRCLE_NODE_INDEX in
+              0)
+                bundle exec rails test test/functional/webui/spider_test.rb -n test_spider_anonymously
+                echo "Test loading the application in production"
```
Comment (dmarcoux) [inline .circleci/conditional_config.yml:343]: As I already told you @hennevogel, I find this weird to test this in a job (`spider` in this case) which has nothing to do with testing the loading of the application. At the same time, having an extra job only for this is a bit overkill.

**Review by dmarcoux — APPROVED** [review]

File: .circleci/conditional_config.yml (hunk 2 above)
Comment (hennevogel) [reply .circleci/conditional_config.yml:343]: indeed

**Pull request merged**